src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs

```diff
@@ -53,31 +53,13 @@ public class NewAzureCognitiveServicesAccountCommand : CognitiveServicesAccountB
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Cognitive Services Account Type.")]
         [Alias(CognitiveServicesAccountTypeAlias, AccountTypeAlias, KindAlias)]
-        [ValidateSet(
-            AccountType.ComputerVision,
-            AccountType.Emotion,
-            AccountType.Face,
-            AccountType.LUIS,
-            AccountType.Recommendations,
-            AccountType.Speech,
-            AccountType.TextAnalytics,
-            AccountType.WebLM,
-            IgnoreCase = true)]
```

Comment:
@felixwa our StaticAnalysis tool is flagging this as a breaking change, which it is not, so I will need to update that and have you pull those changes so this doesn't get incorrectly flagged.